### PR TITLE
solution

### PR DESCRIPTION
**Pull request by savchuksavchuk**
[DEMO LINK](https://savchuksavchuk.github.io/react_tabs/)

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -10,33 +11,20 @@ export const tabs = [
...
 export const App: React.FC = () => {
+  const [selectedTab, setSelectedTab] = useState(tabs[0]);
+
   return (
     <div className="section">
       <h1 className="title">
-        Selected tab is Tab 1
+        Selected tab is
+        {' '}
+        {selectedTab.title}
```
Comment (polosanya) [inline src/App.tsx:21]:
```suggestion
        {`Selected tab is ${selectedTab.title}`}
```

File: src/components/Tabs/Tabs.tsx [hunk 2]
```diff
@@ -1 +1,56 @@
...
+              className={classNames({
+                'is-active': selectedTab.id === tab.id,
+              })}
+              data-cy="Tab"
+              key={tab.id}
+            >
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={(event) => {
```
Comment (polosanya) [inline src/components/Tabs/Tabs.tsx:37]: Move it to the separate function

**Review by polosanya — CHANGES_REQUESTED** [review]
Almost done! Please pay attention to my comments

File: src/components/Tabs/Tabs.tsx [hunk 3]
```diff
@@ -1 +1,62 @@
...
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  const selectHandler = (
+    event: React.MouseEvent<HTMLAnchorElement, MouseEvent>,
+    isSelected: Tab,
```
Comment (loralevitska) [inline src/components/Tabs/Tabs.tsx:24]: the variable `is...`  says that there will be a boolean value, and there will be a number, looks confusing. Rename the variable to a more relevant name. Read about [boolean naming](https://dev.to/michi/tips-on-naming-boolean-variables-cleaner-code-35ig) for better understanding

**Review by loralevitska — CHANGES_REQUESTED** [review]
great! need some improvements

File: src/components/Tabs/Tabs.tsx [hunk 4]
```diff
@@ -1 +1,62 @@
...
+              <a
+                href={`#${tab.id}`}
+                data-cy="TabLink"
+                onClick={(event) => selectHandler(event, currentTab, tab)}
+              >
+                {tab.title}
+              </a>
+            </li>
+          ))}
+        </ul>
```
Comment (Anastasiia-Svintsova) [inline src/components/Tabs/Tabs.tsx:54]: Would be great to move the list and item elements to separate files

**Review by Anastasiia-Svintsova — APPROVED** [review]

**Pull request closed without merging**